data/outreachy.yml

```
@@ -0,0 +1,138 @@
+rounds:
+  - name: Summer 2019
+    projects:
+      - title: "Test the OCaml compiler with code coverage tools"
+        description: ""
+        mentee: "Oxana Kostikova"
+        blog: "https://lereenadem.wixsite.com/personal"
+        mentors:
+         - Sébastien Hinderer
+         - Florian Angeletti
+      - title: "Test the OCaml compiler with random tests and a reference interpreter"
+        description: >
+          The aim of this project is to extend an existing testcase-generator for the OCaml compiler,
+          using a reference interpreter (existing or newly developed) to find a lot of bugs in the compiler,
+          and fix as much of them as possible.
+        mentee: "Ulugbek Abdullaev"
+        blog: "https://ulugbekna.github.io"
+        mentors:
+         - Gabriel Scherer
```

```suggestion
      - title: "Test the OCaml compiler with random tests and a reference interpreter"
        description: >
          The aim of this project is to extend an existing testcase-generator for the OCaml compiler,
          using a reference interpreter (existing or newly developed) to find a lot of bugs in the compiler,
          and fix as much of them as possible.
        mentee: "Ulugbek Abdullaev"
        blog: "https://ulugbekna.github.io"
        mentors:
         - Gabriel Scherer
         - Jan Midtgaard
```
I'm just adding Jan here (see conversation above).